problems/2.functions.js

```diff
@@ -29,7 +34,10 @@ function incrementGlobalCount() {
 * @param {number} thirdNum - The third number
 * @returns {number} - The product of the three numbers
 */
-function multiply() {
+function multiply(firstNum, secondNum, thirdNum) {
+let multiply = (firstNum * secondNum * thirdNum);
+return sum;
```

Comment:
Close. line 38 should have be let sum = 
Because you're retuning sum. mulitply is the name of your function so probably don't name a variable with the same name.